scheduler/plugin/plugin.go

```diff
@@ -191,21 +191,41 @@ func (pl *wasmPlugin) EventsToRegister() (clusterEvents []framework.ClusterEvent
 var _ framework.PreFilterExtensions = (*wasmPlugin)(nil)
 
 // AddPod implements the same method as documented on framework.PreFilterExtensions.
-func (pl *wasmPlugin) AddPod(ctx context.Context, state *framework.CycleState, podToSchedule *v1.Pod, podInfoToAdd *framework.PodInfo, nodeInfo *framework.NodeInfo) *framework.Status {
+func (pl *wasmPlugin) AddPod(ctx context.Context, state *framework.CycleState, podToSchedule *v1.Pod, podInfoToAdd *v1.Pod, nodeInfo *framework.NodeInfo) (status *framework.Status) {
 	// We implement PreFilterExtensions with FilterPlugin, even when the guest doesn't.
 	if pl.guestInterfaces&iPreFilterExtensions == 0 {
 		return nil // unimplemented
 	}
-	panic("TODO: scheduling: AddPod")
+
+	// Add the stack to the go context so that the corresponding host function
+	// can look them up.
+	params := &stack{pod: podToSchedule, podInfo: podInfoToAdd, node: nodeInfo.Node()}
```

Comment:
```suggestion
	params := &stack{pod: podToSchedule, podInfo: podToAdd, node: nodeInfo.Node()}
```

Reply:
@sanposhiho 
Thank you so much for your feedback :bow: